### PR TITLE
Create Localization.md

### PR DESCRIPTION
**Pull request by descorp**
# Open PR

## Changes

* add Localization.md

### CODE REVIEW

File: docs/Localization.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Localization
+
+By default, React Native library uses a device's local. To enable necessary translations for iOS, make sure “Localizations” in the project configuration contains all required languages.
```
Comment (mattlies) [inline docs/Localization.md]:
```suggestion
By default, the React Native library uses a device's locale. To enable necessary translations for iOS, make sure “Localizations” in the project configuration contains all required languages.
```

File: docs/Localization.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## Enforcing specific localization
+
+### iOS
+
+Not supported.
+
+### Android
+
+Provide specific locale to `configuration.shopperLocale` for `AdyenCheckout`.
```
Comment (mattlies) [inline docs/Localization.md]:
```suggestion
Provide a specific locale to `configuration.shopperLocale` for `AdyenCheckout`.
```

File: docs/Localization.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+## Overriding default values
+
+### iOS
+
+1. Open your iOS folder in Xcode.
+2. Create new ’Strings’ file with the name “Localizable”. If you are using multiple localisations - make sure you check-in them all for Localizations.string in "File Inspector": your iOS project will have correspondent files for each localization: `(localization).lproj/Localizable.string`.
```
Comment (mattlies) [inline docs/Localization.md]:
```suggestion
2. Create a new ’Strings’ file with the name `Localizable`. If you are using multiple localizations, make sure you check them all in for the `Localizations.string` in "File Inspector". For each localization, your iOS project will have a corresponding file: `(localization).lproj/Localizable.string`.
```

File: docs/Localization.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+3. Override all necessary strings with desired values for all necessary localizations. A list of available strings could be found at https://github.com/Adyen/adyen-ios/blob/develop/Adyen/Assets/Generated/LocalizationKey.swift.
```
Comment (mattlies) [inline docs/Localization.md]:
```suggestion
3. Override all necessary strings with desired values for all your localizations. The list of available strings can be found at https://github.com/Adyen/adyen-ios/blob/develop/Adyen/Assets/Generated/LocalizationKey.swift.
```

File: docs/Localization.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+### Android
+
+1. Open /res/values/strings.xml in "Translations Editor” in Android Studio.
+2. Override all necessary strings with desired values for all necessary localizations. A list of available strings could be found at https://github.com/search?q=repo%3AAdyen%2Fadyen-android+values%2Fstrings.xml&type=code
```
Comment (mattlies) [inline docs/Localization.md]:
```suggestion
2. Override all necessary strings with desired values for all your localizations. The list of available strings can be found at https://github.com/search?q=repo%3AAdyen%2Fadyen-android+values%2Fstrings.xml&type=code
```

File: docs/Localization.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+## Adding new localization
+
+Add new locale in Xcode and Android Studio respectively. Provide a translation for all necessary keys
```
Comment (mattlies) [inline docs/Localization.md]:
```suggestion
Add new locales in Xcode and Android Studio respectively. Provide a translation for all necessary keys.
```

File: docs/Localization.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+1. Open your iOS folder in Xcode.
+2. Create new ’Strings’ file with the name “Localizable”. If you are using multiple localisations - make sure you check-in them all for Localizations.string in "File Inspector": your iOS project will have correspondent files for each localization: `(localization).lproj/Localizable.string`.
+3. Override all necessary strings with desired values for all necessary localizations. A list of available strings could be found at https://github.com/Adyen/adyen-ios/blob/develop/Adyen/Assets/Generated/LocalizationKey.swift.
+
+### Android
+
+1. Open /res/values/strings.xml in "Translations Editor” in Android Studio.
+2. Override all necessary strings with desired values for all necessary localizations. A list of available strings could be found at https://github.com/search?q=repo%3AAdyen%2Fadyen-android+values%2Fstrings.xml&type=code
+
+## Adding new localization
```
Comment (mattlies) [inline docs/Localization.md]:
```suggestion
## Adding new localizations
```

File: docs/Localization.md (hunk 6 above)
Comment (descorp) [reply docs/Localization.md]: Fixed in [3d928ec](https://github.com/Adyen/adyen-react-native/pull/165/commits/3d928ec9faeb29ab435a3a2b963190e4e9a1374f)

**Review by jreij — APPROVED** [review]

**Pull request merged**